ProcessLib/UncoupledProcessesTimeLoop.h

```diff
@@ -142,6 +149,7 @@ std::unique_ptr<UncoupledProcessesTimeLoop> createUncoupledProcessesTimeLoop(
     std::map<std::string, std::unique_ptr<Process>> const& processes,
     std::map<std::string, std::unique_ptr<NumLib::NonlinearSolverBase>> const&
         nonlinear_solvers,
-    std::vector<std::unique_ptr<MeshLib::Mesh>> const& meshes);
+    std::vector<std::unique_ptr<MeshLib::Mesh>> const& meshes,
+    std::unique_ptr<ProcessLib::ChemistryLib::phreeqcIO>& phreeqcIO);
```

Comment:
passing a writable reference to a unique_ptr looks odd.

Reply:
PhreeqcIO is a dataset holder for data communication between OGS and Phreeqc. In each time step, the dataset holder needs to be either reset or updated before and after chemical calculation by Phreeqc. Therefore, the unique_ptr to class PhreeqcIO is not constfied.